core/net/mac/tsch/tsch-queue.h

```diff
@@ -39,6 +39,7 @@
 #include "lib/ringbufindex.h"
 #include "net/linkaddr.h"
 #include "net/mac/tsch/tsch-schedule.h"
+#include "net/mac/mac.h"
```

Comment:
Hi Simon,
the code in `tsch-queue.h` file depends on the declaration of `mac_callback_t`, which may lead to compilation error for applications that directly include `tsch-queue.h` before `net/mac/mac.h` header. If the file is private and should not be included directly by applications maybe mark it as such.